frontend/src/routes/(authenticated)/admin/+page.svelte

```diff
@@ -80,20 +71,20 @@
 </svelte:head>
 <main>
   <div class="grid lg:grid-cols-2 grid-cols-1 gap-10">
-    <ProjectTable bind:this={projectsTable} projects={$allProjects} users={$allUsers?.items ?? []} {defaultFilterLimit}/>
+    <ProjectTable bind:this={projectsTable} projects={$allProjects} />
 
     <div>
       <span class="text-xl flex gap-4">
         {$t('admin_dashboard.user_table_title')}
         <Badge>
           <span class="inline-flex gap-2">
-            {userSearch ? filteredUsers.length : users.length}
+            {showUsers.length}
             <span>/</span>
-            {$allUsers?.totalCount}
+            {totalUsers}
           </span>
         </Badge>
       </span>
-      <Input label="" placeholder={$t('admin_dashboard.filter_placeholder')} bind:value={userSearch} />
+      <Input label="" placeholder={$t('admin_dashboard.filter_placeholder')} bind:value={$queryParams.userSearch} />
```

Comment:
so I'm not sure what the best way to do this would be, but we need to debounce the binding (or specifically the setting) of the search param here. Otherwise when I type `Kevin` I get 5 gql requests, one for each extra letter, no matter how fast I type or if any other the other requests have even finished.

Reply:
Right. I'm glad you caught that 👍